hackerrank/1-week-prep/day-5/mock-test-5.py

```diff
@@ -0,0 +1,19 @@
+def pairs(exp_difference, values):
+    num_pairs = 0
+    occurrences = dict()
+
+    for value in values:
+        if value not in occurrences:
+            occurrences[value] = 0
+        occurrences[value] += 1
```

Comment:
One of the constraints of the problem is that each `value` in `values` will be unique, so you didn't have to keep track of occurrences.